test/module/libs/crypto/keys_manager_test.cpp

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include <gtest/gtest.h>
+#include <boost/filesystem.hpp>
+#include <fstream>
+#include <string>
+#include "crypto/keys_manager_impl.hpp"
+
+#define PUBKEY_HEX_SIZE 64
```

Comment:
Don't use preprocessor commands if it may be replaced with c++ style code. Use constants instead.